targets/gen_noro_nowcasts_targets.R

```diff
@@ -50,21 +51,62 @@ gen_noro_nowcasts_targets <- list(
   ),
   # Forecast quantiles as a scoringutils forecast object
   tar_target(
-    name = quantiled_nowcast_noro,
+    name = su_quantile_noro,
     command = scoringutils::as_forecast_quantile(
-      data = su_forecast_sample,
-      probs = config$Hub_quantiles
+      data = su_sample_noro,
+      probs = config$norovirus$quantiles
     )
   ),
   # Get a wide dataframe with only 50th and 95th for plotting
   tar_target(
     name = summary_nowcast_noro,
-    command = quantiled_nowcast_noro |>
+    command = su_quantile_noro |>
       filter(quantile_level %in% c(0.025, 0.25, 0.5, 0.75, 0.975)) |>
       pivot_wider(
         names_from = "quantile_level",
         values_from = "predicted",
         names_prefix = "q_"
       ) |> left_join(data_as_of, by = "reference_date")
+  ),
+  # Scores--------------------------------------------------------------------
+  tar_target(
+    name = scores_sample_noro,
+    command = scoringutils::score(su_sample_noro)
+  ),
+  tar_target(
```

Comment:
why duplicating?

Reply:
I was just going to score both samples and quantiles to have both as options